src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+import { Person } from '../types/Person';
+import { PeopleTable } from './PeopleTable';
+import { getPeople } from '../api';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [activePerson, setActivePerson] = useState<Person | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+
+  useEffect(() => {
+    setIsLoading(true);
+    setError(null);
+
+    getPeople()
+      .then((data: Person[]) => {
+        const enhancedPeople = data.map((person: Person) => ({
+          ...person,
+          mother:
+            data.find((p: Person) => p.name === person.motherName) || undefined,
+          father:
+            data.find((p: Person) => p.name === person.fatherName) || undefined,
```

Comment:
The mapping logic for `mother` and `father` assumes that `motherName` and `fatherName` are available in the `Person` type. Ensure these properties exist and are correctly used to find the corresponding parent objects.